README.md

```diff
@@ -91,6 +91,28 @@ Try the workshop at https://apprunnerworkshop.com
 Read the docs at https://docs.aws.amazon.com/apprunner
 ```
 
+## Table of Contents
```

Comment:
GitHub shows a ToC by default. I'm worried that the manually created one goes out of sync eventually
<img width="1055" alt="Screenshot 2025-01-02 at 18 44 36" src="https://github.com/user-attachments/assets/6a01b000-22ed-4df6-9536-8eb4169f8a64" />